src/apps/activity_assignments/service.py

```diff
@@ -136,3 +205,11 @@ async def _get_assignments_entities(
         }
 
         return entities
+
+    @staticmethod
+    def _get_email_template_name(language: str) -> str:
+        return f"new_activity_assignments_{language}"
+
+    @staticmethod
+    def _get_email_assignment_subject(applet: AppletSchema):
+        return "Assignment Notification"
```

Comment:
Shouldn't this be internationalized – something like the below? (Or using whatever is the correct way to localize text in Python…)

```suggestion
    def _get_email_assignment_subject(applet: AppletSchema, language: str) -> str:
        translations = {
            "en": "Assignment Notification",
            "fr": "Notification d'attribution",
        }
        return translations.get(language, translations["en"])
```

Reply:
done